### PR TITLE
[JA] Add `Ranking Criteria/osu!catch`

### PR DESCRIPTION
**Pull request by Usakitsu**
per the catch meeting the community held a couple months back, we wanted the ranking criteria translated into more of the popular languages. i had it reviewed by -rei, mentai, and resona (non-natives) and by amulet-pp, a japanese native catch mapper all not through github (on discord instead). i think it should be good to go but anyone is welcome to add their suggestions on wording or grammar

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Review by peppy — CHANGES_REQUESTED** [review]

File: wiki/Ranking_criteria/osu!catch/ja.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
+# osu!catch ranking criteria
+
+***注目：この記事は[一般的なランキング基準](/wiki/Ranking_criteria)の拡張版です。***
+
+osu!catch ranking criteria(osu!catch譜面のランク基準)はosu!catch譜面を作成する際に従わなければならない[ルール、ガイドライン](/wiki/Ranking_criteria#general-terms)です。osu!catch譜面をRankedさせるためにはこれらのルールに従う必要があります。いかなる場合でもルールに従わなければなりませんが、ガイドラインについては例外的な状況下において無視することが出来ます。ただし、その場合は何故ガイドラインを無視しないとosu!catch譜面全体の品質に影響を及ぼすのか、具体的な説明が必要となります。
```
[the remark this answers is not in the document]
Comment (Usakitsu) [reply wiki/Ranking_criteria/osu!catch/ja.md:5]: i think i copied this over when i was just starting out and looking at the japanese version of the taiko ranking criteria for inspiration, i thought this was the way it was wanted but now going back and seeing the outdated tag thats probably not the case lol. i've now rewritten it to match back the original

File: wiki/Ranking_criteria/osu!catch/ja.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+- **hitnormalの付いていない[キーサウンド](/wiki/Beatmapping/Hitsound#keysound)を使用してはいけません。** サウンドが曲と完全に調和していると、打時のフィードバックが聞き取りにくいためです。
+- **もし、各難易度の[drain time](/wiki/Beatmap/Drain_time)が…**
+  - **…2分30秒を下回る場合**、 最低難易度はSaladより難しくできません。
+  - **…2分30秒から3分15秒の間の場合**、 最低難易度はPlatterより難しくできません。
+  - **…3分15秒から4分の間の場合**、 最低難易度はRainより難しくできません。
+  - **上記の閾値を満たすために[drain time](/wiki/Beatmap/Drain_time)と[break time](/wiki/Beatmap/Break)組み合わせることができます。** 最高難易度の場合は、break timeは最大30秒に制限されます。drain timeが30秒以内の譜面は適用されません。
+
+#### ガイドライン (Guidelines)
+
+- **全てのサークルとスライダーは曲の特徴的な音に合わせてください。** 通常は明瞭な音ですが、始まりと終わりの区別がつかない連続音を表すこともあります。
```
[the remark this answers is not in the document]
Comment (Usakitsu) [reply wiki/Ranking_criteria/osu!catch/ja.md:26]: i don't think so? its supposed to be representing "This is usually a distinct sound, but can also represent a continuous sound with an indistinguishable start or end." but correct me if im wrong

File: wiki/Ranking_criteria/osu!catch/ja.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+- **カスタムオブジェクトはすべての必要な要素を含み、グレーの色で着色されている必要があります。** これは画像がはっきりと表示され、許容できる品質にするためです。必要な要素は[Skinning/osu!catch](/wiki/Skinning/osu!catch)で見つけることができます。さらに、オーバーレイには透明な要素を使用することを推奨します。
+- **スキン要素は、デフォルトスキンの対応要素と同じサイズである必要があります。** これは適切にヒットボックスを表現し、ゲームプレイが変更されないようにするためです。デフォルトスキンで使用されている現在の寸法は、[フルーツ](/wiki/Gameplay/Hit_object/Fruit)には128x128、[ドロップ](/wiki/Gameplay/Hit_object/Juice_stream#drop)には82x103、キャッチャーには306x320です。
+
+#### ガイドライン (Guidelines)
+
+- **カスタムキャッチャーは、スキンセットを完成させるために`lighting.png`要素を追加することが推奨されます。** 追加は任意であり、デフォルトの寸法は184x184ですが、要素の目的の可視性に応じて異なる場合があります。
+
+## 難易度別 (Difficulty-specific)
+
+難易度別のルールとガイドラインは、それぞれの難易度レベルにのみ適用されます。**したがって、それらはすべてのosu!catch難易度に適用されるわけではありません。** リズムに関連するルールとガイドラインは4/4拍子でBPM 180程度の曲に基づいています。曲が大幅に速い、または遅い場合は、[Ranking CriteriaにおけるBPMのスケーリング](/wiki/Ranking_criteria/Scaling_BPM)で詳細に説明されています。
```
[the remark this answers is not in the document]
Comment (Usakitsu) [reply wiki/Ranking_criteria/osu!catch/ja.md]: i just copied it over from the first time this appeared without realizing 4/4 wasn't mentioned, has been fixed

[the remark this answers is not in the document]
Comment (Usakitsu) [reply wiki/Ranking_criteria/osu!catch/ja.md]: applied

**Review by peppy — APPROVED** [review]
Seems good 👍

**Pull request merged**